### PR TITLE
fix footer

### PR DESCRIPTION
**Pull request by arivasvera**

### CODE REVIEW

**Review by jMavarez — COMMENTED** [review]
Could you upload a picture of the before & after?

File: messages.js [hunk 1]
```diff
@@ -158,9 +160,11 @@ exports.embedInventoryAsImage = async (event, locale) => {
...
     victim: event.Victim.Name,
   });
-  const footer = good
-    ? l.__("KILL.FOOTER_GOOD", { killer: event.Killer.Name })
-    : l.__("KILL.FOOTER_BAD", { killer: event.Victim.Name });
+  const footer = {
+    text: good
+      ? l.__("KILL.FOOTER_GOOD", { killer: event.Killer.Name })
+      : l.__("KILL.FOOTER_BAD", { killer: event.Victim.Name }),
+  };
```
Comment (jMavarez) [inline messages.js:167]: This can be a function since it's the same implementation as in `embedEventAsImage`.

Comment (jMavarez) [inline messages.js]: This can be a function since it's the same implementation as in `embedEventAsImage`.

**Review by jMavarez — CHANGES_REQUESTED** [review]

**Comment by arivasvera** [comment]
> 
> 
> Could you upload a picture of the before & after?

I can't, but I know it's wrong because I tried it in another instance.
Anyway I share the link of the discord documentation where the format of the footer is detailed:
https://discord.com/developers/docs/resources/channel#embed-object-embed-footer-structure

File: messages.js (hunk 1 above)
Comment (arivasvera) [reply messages.js]: Done

File: messages.js [hunk 2]
```diff
@@ -10,6 +10,15 @@ const RED = 13369344;
 const BATTLE = 16752981;
 const RANKING_LINE_LENGTH = 23;
 
+const getFooter = (killerName, VictimName, locale, good) => {
+  const l = exports.getI18n(locale);
```
Comment (jMavarez) [inline messages.js]:
```suggestion
const getFooter = (killerName, victimName, locale, good) => {
  const l = exports.getI18n(locale);
```

File: messages.js [hunk 3]
```diff
@@ -124,15 +133,9 @@ exports.embedEventAsImage = async (event, locale) => {
     killer: event.Killer.Name,
     victim: event.Victim.Name,
   })}](${KILL_URL.replace("{lang}", l.getLocale()).replace("{kill}", event.EventId)})`;
-  let footer = "";
+  const footer = hasInventory ? null : getFooter(event.Killer.Name, event.Victim.Name, locale, good );
```
Comment (jMavarez) [inline messages.js]:
```suggestion
  const footer = hasInventory ? null : getFooter(event.Killer.Name, event.Victim.Name, locale, good);
```

**Review by jMavarez — APPROVED** [review]

File: messages.js (hunk 2 above)
Comment (arivasvera) [reply messages.js]: done

File: messages.js (hunk 3 above)
Comment (arivasvera) [reply messages.js]: done

**Pull request merged**